### PR TITLE
Review the code 

### PR DESCRIPTION
**Pull request by KyawSwarMin**
Please ko Joseph and Ko D Ther Htun, Please kindly review my code.

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -1,35 +1,65 @@
...
+  manage_default_network_acl    = false
+  manage_default_security_group = false
+
+  tags = {
+    terraform  = true
+    Enviroment = "prod"
+  }
+}
+
+resource "aws_instance" "jdm-bastion-host" { // OS = Ubuntu
```
Comment (josephphyo) [inline main.tf]: @KyawSwarMin , Should be UnderScore.

File: main.tf [hunk 2]
```diff
@@ -1,35 +1,65 @@
...
+  associate_public_ip_address = true
+  subnet_id                   = module.vpc.public_subnets[0]
+  vpc_security_group_ids      = [aws_security_group.jdm-public-sg.id]
+
+  tags = {
+    Name = "${var.project}-bastion-host"
+  }
+}
+
+resource "aws_instance" "Kube-control" { // OS = Amazon OS
```
Comment (josephphyo) [inline main.tf]: @KyawSwarMin , Should be UnderScore.

File: main.tf [hunk 3]
```diff
@@ -1,35 +1,65 @@
...
+  associate_public_ip_address = true
+  subnet_id                   = module.vpc.private_subnets[0]
+  vpc_security_group_ids      = [aws_security_group.jdm-private-sg.id]
+
+  tags = {
+    Name = "${var.project}-Kube-control"
+  }
+}
+
+resource "aws_instance" "Kube-worker" { // OS = Amazon OS
```
Comment (josephphyo) [inline main.tf]: @KyawSwarMin , Should be UnderScore.

File: main.tf (hunk 1 above)
Comment (KyawSwarMin) [reply main.tf]: @josephphyo Done ako

**Comment by KyawSwarMin** [comment]
Can you approve?

**Review by josephphyo — APPROVED** [review]

**Pull request merged**